Backend/prisma/migrations/20231207130626_init/migration.sql

```diff
@@ -0,0 +1,16 @@
+-- CreateTable
+CREATE TABLE "users" (
+    "id" UUID NOT NULL,
+    "emai" VARCHAR(255) NOT NULL,
+    "name" VARCHAR(255) NOT NULL,
+    "phone" VARCHAR(255),
```

Comment:
for phone maximum we need is 30. never need more than that
len("00963 (123) 1234 123456678") =26